.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+  # Run tests against the forked branch, but
+  # do not allow access to secrets
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflows-in-forked-repositories
+  pull_request:
+
+env:
+  SDK_VERSION: "7.0.x"
+  TEST_ARGS: >
+    --configuration Release
+    --collect:"XPlat Code Coverage"
+    --settings tests/coverletArgs.runsettings
+    --verbosity minimal
+
+
+jobs:
+  run-tests:
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+
+    runs-on: "${{ matrix.os }}"
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.SDK_VERSION }}
+
+      - name: Run DotNet CLI Tests
+        run: >
+          find tests -type f -iname "*.csproj" -print0 | xargs -0 -I{} -t
```

Comment:
Our Azure pipeline uses the `"tests/**/*Tests.csproj"` pattern.

Reply:
This find looks only in `tests` (recursively for `*.csproj`), it's case insensitive (to unify Windows and POSIX), so the equivalent shell glob would be: `tests/**/*.csproj` 

but I'm happy to tweak it back to the original equivalent still using find?

`find tests -type f -name "*Tests.csproj" -print0`